frontend/src/context/anticipatoryAction/AAStormStateSlice/utils.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  AACategory,
+  AACategoryDataToLandfallMap,
+  AACategoryKey,
+  AACategoryKeyToCategoryMap,
+  AACategoryLandfall,
+  DistrictDataType,
+  MergedFeatures,
+  ResultType,
+} from './parsedStormDataTypes';
+import { StormDataResponseBody, WindState } from './rawStormDataTypes';
+
+const districtNameMapping: { [key: string]: string } = {
+  Maganja_Da_Costa: 'Maganja Da Costa',
+  Cidade_Da_Beira: 'Cidade Da Beira',
+};
+
+const watchedDistricts: { [key in AACategory]: string[] } = {
+  [AACategory.Severe]: [
```

Comment:
We should probably move away from the Severe/Moderate nomenclature used for AADrought since the meaning doesn't really match in this case.